superset/assets/src/explore/controlPanels/Area.js

```diff
@@ -0,0 +1,48 @@
+import { t } from '@superset-ui/translation';
+import * as sections from './sections';
```

Comment:
since we're doing a big refactor, could we import just what's used in this file? `*` is harder for readability + linting, etc.

Reply:
sgtm. Just pushed again with specific imports